WalletWasabi.Gui/ManagedDialogs/ManagedFileChooserFilterViewModel.cs

```diff
@@ -8,7 +8,7 @@ namespace WalletWasabi.Gui.ManagedDialogs
 {
 	internal class ManagedFileChooserFilterViewModel : ViewModelBase
 	{
-		private readonly string[] _extensions;
+		private readonly string[] Extensions;
```

Comment:
In fact it could be `private IEnumerable<string> Extensions { get; }`.

Reply:
Done.